utils/unittest/fixtures.go

```diff
@@ -2643,7 +2643,7 @@ func ChunkExecutionDataFixture(t *testing.T, minSize int, opts ...func(*executio
 
 // RootProtocolStateFixture creates a fixture with correctly structured data for root protocol state.
 // This can be useful for testing bootstrap when there is no previous epoch.
-func RootProtocolStateFixture() *flow.RichProtocolStateEntry {
+func RootProtocolStateFixture() *flow.RichEpochProtocolStateEntry {
```

Comment:
```suggestion
// RootEpochProtocolStateFixture creates a fixture with correctly structured Epoch sub-state.
// The epoch substate is part of the overall protocol state (KV store). 
// This can be useful for testing bootstrap when there is no previous epoch.
func RootEpochProtocolStateFixture() *flow.RichEpochProtocolStateEntry {
```

Reply:
https://github.com/onflow/flow-go/pull/5968/commits/8cdfb26f3c19260a7dbf5dc1351b574ee8f0ff1b